transformations/trim.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package transformations
+
+import "strings"
+
+/*
```

Comment:
There is a huge mix of comments in the project, if we are going to standardize it lets create a different PR

Reply:
But you are adding this just now, why do we want to increase the confusion? Can't we just do the right thing here and then the PR that will standardize this will be smaller.